kubespawner/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+JupyterHub Spawner to spawn user notebooks on a Kubernetes cluster.
+
+After installation, you can enable it by adding:
+
+```
+c.JupyterHub.spawner_class = 'kubespawner.KubeSpawner'
+```
+
+in your `jupyterhub_config.py` file.
+
+We export KubeSpawner specifically here, so people can import kubespawner.KubeSpawner
```

Comment:
We export KubeSpawner specifically here. This simplifies import for users. Users can simply `import kubespawner.KubeSpawner` in their applications instead of the more verbose `import kubespawner.spawner.KubeSpawner`.